docker/Dockerfile-debug

```diff
@@ -0,0 +1,70 @@
+FROM ubuntu:22.04
```

Comment:
If you have some notes on how to use this for developing the plugin it would be very nice to add it in a .md file